lib/VMwareWebService/MiqVimVm.rb

```diff
@@ -721,7 +721,7 @@ def resetCd
 	# If backingFile is just the datastore name, "[storage 1]" for example,
 	#    file names will be generated as appropriate.
 	#
-	def addDisk(backingFile, sizeInMB, label=nil, summary=nil, thinProvisioned=false)
+	def addDisk(backingFile, sizeInMB, label=nil, summary=nil, thinProvisioned=false, dependent=false, persistent=true)
```

Comment:
Now that we've added additional parameters, and may need to add more in the future, I think we should add hashed parameters after `summary`.